kraken/kubernetes/client.py

```diff
@@ -281,20 +285,18 @@ def get_job_status(name, namespace="default"):
         raise
 
 
-# Obtain node status
-def get_node_status(node, timeout=60):
-    try:
-        node_info = cli.read_node_status(node, pretty=True, _request_timeout=timeout)
-    except ApiException as e:
-        logging.error(
-            "Exception when calling \
-                       CoreV1Api->read_node_status: %s\n"
-            % e
-        )
-        return None
-    for condition in node_info.status.conditions:
-        if condition.type == "Ready":
-            return condition.status
+# Obtain the node status
+def get_node_status(node, status, timeout):
```

Comment:
Are we using this function now? Seems to be doing  almost the same thing as watch_node_status.

Reply:
I've removed this function in favour of `watch_node_status`.